### ISSUE TITLE
MueLu: Consider allowing "long long" for Tpetra GO when building MueLu tutorial

### ISSUE THREAD

**Issue by abodado** [trilinos/Trilinos#13500]
When building Trilinos, I often see this message:

> -- MueLu's tutorial requires for a correct build that if Tpetra is enabled, then GlobalOrdinal=int must be enabled, and either the Serial or OpenMP Nodes in Tpetra be enabled.  Since this is not the case, I am disabling MueLu's tutorial by default.  See Trilinos GitHub Issue #1726 for details: https://github.com/trilinos/Trilinos/issues/1726
> -- You chose to enable MueLu's tutorial, even though I told you that it may not build correctly.  You are responsible for whatever build or link errors may occur as a result.
> 

Not an urgent matter, but am curious if plausible to update the MueLu example source code to allow the tutorial to be built with "long long" GO for Tpetra, since "long long" is Tpetra's default GO.

**Comment by jhux2** [comment]
@GrahamBenHarper @mayrmt Is this something that is going to be addressed by the tutorial update?

**Comment by mayrmt** [comment]
@jhux2 yes, the tutorial update will migrate all examples to Tpetra, so this should be easy to fix. Current behavior arises from the old Epetra examples.

**Comment by mayrmt** [comment]
@PhilipOesterlePekrun, can you confirm that PR #14357 addresses this issue?

**Comment by PhilipOesterlePekrun** [comment]
@mayrmt Yes, all the ctests work with the new changes with
```
-D Trilinos_ENABLE_Tpetra:BOOL=ON \
-D Tpetra_INST_INT_INT:BOOL=OFF \
-D Tpetra_INST_INT_LONG_LONG:BOOL=ON \
-D Trilinos_ENABLE_Epetra=OFF \
-D Trilinos_ENABLE_EpetraExt=OFF \
```
on all of my available Kokkos backends. And the warning is no longer there of course.
